### DISCUSSION TITLE
PAGA analysis

### DISCUSSION THREAD

**Discussion by zhouxuan-njmu-nmu** [zhanghao-njmu/SCP#174] [Q&A]
Dear author, when i run the PAGA analysis, it seems to have trouble.

 `experiment.aggregate <- RunPAGA(
  srt = experiment.aggregate, group_by = "celltype",
  linear_reduction = "PCA", nonlinear_reduction = "UMAP"
)`
 " Error in py_module_import(module, convert = convert) : 
  NameError: name 'base_events' is not defined ".

It seems to be related with python code?

**Comment by zhanghao-njmu** [comment]
The same issue as in #172
